Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 members = [
   "jormungandr-utils",
+  "jormungandr-lib",
```

Comment:
Is this different in purpose from jormungandr-utils so that this crate proliferation is justified?

Reply:
Alternatively, -utils could be morphed into this.

Reply:
it is going to replace it.

Reply:
it was stated in the commit message of the first commit, but not repeated in the PR description:

> add introduce jormungandr-lib, a set of components to replace jormungandr-utils

Reply:
I'd just do it right away then.

Reply:
too much code change/churn at once